pkg/grpc/api.go

```diff
@@ -1524,7 +1524,8 @@ func (a *api) UnsubscribeConfiguration(ctx context.Context, request *runtimev1pb
 	_, ok := a.CompStore.GetConfigurationSubscribe(subscribeID)
 	if !ok {
 		return &runtimev1pb.UnsubscribeConfigurationResponse{
-			Ok: true,
+			Ok:      false,
```

Comment:
i think this is not a breaking change in the api. Rather a bugfix. 
cc @yaron2 @artursouza @daixiang0

Reply:
Yeah http api is already working as expected. Just in grpc, it required the fix.

Reply:
Agree.